public/css/backend.css

```diff
@@ -17069,3 +17069,6 @@ body {
   padding-bottom: 1rem;
 }
 
+.white-font {
```

Comment:
Directly modifying the compiled css will result in it being deleted whenever laravel-mix recompiles the public assets.

Also, there already exists a css class to specify white text: `.text-white`